runtime/caml/startup_aux.h

```diff
@@ -18,6 +18,8 @@
 
 #ifdef CAML_INTERNALS
 
+#include <stdbool.h>
```

Comment:
Is including `stdbool.h` fine here w.r.t (the absence of) namespaces and C++ ABI compatibility because it's guarded by `CAML_INTERNALS`?

Reply:
Instinctively I'd say yes, but is there an immediately cleaner way I could put it? The other option would be just to use an `int`, of course 😉

Reply:
I think it's fine, we're not guarding things under `CAML_INTERNALS` with `extern "C"` for C++ compat either.